src/v2/components/Link.js

```diff
@@ -19,8 +19,11 @@ const Link = View.extend({
   tagName: 'a',
 
   attributes () {
-    const customHelpLink = this.options.settings.get('helpLinks.help');
-    let href = customHelpLink ? customHelpLink: this.options.href || '#';
+    if (this.options.href && !this.options.customLink) {
```

Comment:
looks like you could construct the `href` (prefix base url) from `FormBuilder` so that logic in `Link.js` could be simplified as
` if this.options.href then use it otherwise '#'`.